network.go

```diff
@@ -208,7 +216,7 @@ type network struct {
 	attachable   bool
 	inDelete     bool
 	ingress      bool
-	driverTables []string
+	driverTables []networkDBTable
```

Comment:
Since there are no changes to the external APIs I don't think there is any backward compatibility concern.